safe/utilities/rounding.py

```diff
@@ -13,15 +13,20 @@
 __revision__ = '$Format:%H$'
 
 
-def format_number(x, enable_rounding=True, coefficient=1):
+def format_number(
+        x, enable_rounding=True, is_population=True, coefficient=1):
```

Comment:
is_population should be False by default I think. But I don't want to break tests in reporting for now.